packages/strapi-plugin-content-manager/admin/src/containers/ListView/ListViewLayout.js

```diff
@@ -0,0 +1,49 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+import { setLayout } from './actions';
+import ListView from '.';
+import { useQueryParams } from '../../hooks';
+
+const ListViewLayout = ({ layout, ...props }) => {
+  const dispatch = useDispatch();
+  const initialParams = useSelector(state => state.get('content-manager_listView').initialParams);
+  // eslint-disable-next-line no-unused-vars
+  const [_, setQuery] = useQueryParams(initialParams);
```

Comment:
`const [, setQuery] = useQueryParams(initialParams);`

So eslint does not complain